pkg/util/scrubber/default.go

```diff
@@ -316,6 +316,16 @@ func ScrubLine(url string) string {
 	return DefaultScrubber.ScrubLine(url)
 }
 
+// HideKeyExceptLastFiveChars replaces all characters in the key with "*", except
+// for the last 5 characters. If the key is an unrecognized length, replace
+// all of it with the default string of "*"s instead.
+func HideKeyExceptLastFiveChars(key string) string {
+	if len(key) != 32 && len(key) != 40 {
+		return strings.Repeat("*", len(key))
```

Comment:
Should we use `defaultReplacement` as maxime suggested?

Reply:
Oops, yes that should be used here, my mistake.